rmgpy/solver/liquid.pyx

```diff
@@ -54,15 +54,16 @@ cdef class LiquidReactor(ReactionSystem):
     cdef public ScalarQuantity P    
     cdef public double V
     cdef public bint constantVolume
+    cdef public double viscosity
```

Comment:
Why is this necessary in this commit?

Reply:
It's not. Should have been part of the liquid phase commit, but somehow I lost that change. Will update.

Reply:
Moved to other commit.